package.json

```diff
@@ -29,7 +29,6 @@
   },
   "devDependencies": {
     "@gimenete/type-writer": "^0.1.5",
-    "@octokit/core": "^3.0.0",
```

Comment:
please keep the dev dependency. I think since npm 7, peer dependencies are installed automatically when needed, but I want to decouple the range of the peer dependency of the range from the dev dependency